_kbarticles/2025-03-27-monitoring-app-metrics-opensearch.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "Monitoring your application's health using OpenSearch"
+date: March 27, 2025
+excerpt: How to monitor the CPU and memory metrics for your application on Cloud.gov using OpenSearch
+---
+
+One of the benefits of hosting your applications on Cloud.gov is that it offers [performance metrics for
+monitoring the health of your application](https://docs.cloudfoundry.org/loggregator/container-metrics.html).
+
+The most important metrics for monitoring your application's health are the memory and CPU metrics, which can help you identify:
+
+- If your application has enough memory provisioned per instance
```

Comment:
Put a period at the end of each clause (see rule 3 at https://grammargoddess.com/monday-grammar-checkup-3-simple-rules-for-using-colons-with-lists/)